edx_rest_api_client/auth.py

```diff
@@ -48,3 +48,16 @@ def __call__(self, r):
         """ Update the request headers. """
         r.headers['Authorization'] = 'Bearer {}'.format(self.token)
         return r
+
+
+class OidcIdTokenAuth(AuthBase):
```

Comment:
At the very least give this class the correct name (e.g. `GeneratedJwtAuth`). This has nothing to do with OpenID or OpenID Connect.